### DISCUSSION TITLE
Please provide your Apple Developer Program account credentials

### DISCUSSION THREAD

**Discussion by xieyezi** [fastlane/fastlane#18488] [Q&A]
We work fastlane with Jenkins,But as seem as fastlane is not able to access the keychain from Jenkins. I have verified that the correct credentials are in the keychain. The credentials are added via the keychain manager.

And If I run the same command in Terminal without Jenkins , fastlane work well.  I am not sure what is going on.

![image](https://user-images.githubusercontent.com/16821989/113083807-73ce5980-920f-11eb-9119-65f55eb46401.png)

**Comment by xieyezi** [comment]
I also have add apple session  by `fastlane spaceauth -u apple@id`
this is my jenkins excute shell:
```shell
flutter build ios --release -v
cd ios
export FASTLANE_SESSION=$(cat login_cookie)
fastlane ad_hoc
ls Runner.*
```

And this is my fastfile:
```
default_platform(:ios)

platform :ios do
  desc "Description of what the lane does"
  lane :ad_hoc do
    # add actions here: https://docs.fastlane.tools/actions
    match(
      type: "adhoc",
      app_identifier: "xxxxx"
    )
    build_ios_app(
      clean: true
    )
  end
end

```

**Reply by xieyezi** [reply]
I have resolve it

**Reply by caleb-harrelson** [reply]
> I have resolve it

How? Sharing your solution may help others in a similar bind.
